qml/Stage/Stage.qml

```diff
@@ -91,19 +91,13 @@ FocusScope {
 
     onInteractiveChanged: {
         // Stage must have focus before activating windows, including null
-        if (interactive) {
-            focus = true;
+        if (interactive) focus = true;
```

Comment:
Better I think to leave the code here as was, and just close off the if statement early with closing brace on a new line.